docs/edgeql/operators/type.rst

```diff
@@ -35,8 +35,54 @@ IS
         ... FILTER User.name = 'Alice';
         {True}
 
-        db> SELECT User IS (Text, Named);
-        {True, ..., True}  # one for every user instance
+.. eql:operator:: TYPEUNION: A | B
+
+    :optype A: object type
+    :optype B: object type
+    :resulttype: union type
+    :index: union type
+
+    Create a *union type* from types ``A`` and ``B``.
+
+    A *union type* is a way to describe the type of a set resulting from a
+    :eql:op:`UNION` of two or more unrelated object types. This
+    expression can only be used on the right-hand-side of
+    :eql:op:`IS` or as the :ref:`concrete link
+    <ref_datamodel_links_concrete>` target type.
+
+    Consider the *union type* ``Text | Named``. Every ``User`` is of
+    that *union type* (because ``User`` extends ``Named``):
```

Comment:
I think the emphasis is only necessary in the first occurrence of the term.

Reply:
I wanted to emphasize it consistently in this section.